messaginginapp/src/sharedTest/java/io/customer/messaginginapp/ModuleMessagingInAppTest.kt

```diff
@@ -31,6 +34,13 @@ internal class ModuleMessagingInAppTest : BaseTest() {
 
         module.initialize()
 
+        // verify gist is initialized
         verify(gistInAppMessagesProvider).initProvider(any(), eq("test"))
+
+        // verify hook was added
+        verify(hooksManager).add(any(), any())
+
+        // verify events
+        verify(gistInAppMessagesProvider).subscribeToEvents(any(), any(), any())
```

Comment:
I am ok with this test function testing for multiple expected behaviors. 
1. The gist SDK is initialized
2. Module hooks initialized. 
3. Module listening for Gist events. 

I think it would be good to either (1) update the name of the test function to reflect all 3 things being tested against in the test function or (2) split up this test function into 3 separate test functions where each test function tests 1 of the 3 things above.